hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,35 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+<a name="Raspberry Pi 3B+, CM3+"></a>
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is the same [procedure](../../../installation/installing-images/installation/installing-images/) as for SD cards.
+
+On Windows you may have to initialise a (clean) harddrive before it's visible and has a drive letter (which may be required to write the image).
+
+After preparing the storage device, connect the drive to the Raspberry Pi and power up the Pi (be aware of the extra USB power usage from external drive).
+After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
+
+<a name="unsupported devices"></a>
+## Unsupported devices
+
+Including Raspberry Pi 1, 2 (V1.1), Compute Module, Zero.
+
+The boot code for USB is stored in the BCM2837(B0) device only. The Pi 1 A/B, Pi 2 B (v1.1), and Pi Zero will all require SD cards as they are based on the BCM2835 and BCM2836. This boot code for USB is stored in ROM (except Pi 4B) which by definition cannot be changed.
```

Comment:
Hmm, what about something like
"The boot code for USB Mass Storage is present in the (non-updateable) ROM of the BCM2837 devices only. The Pi 1 A/B/A+/B+, Pi 2 B (v1.1), and Pi Zero are based on the earlier BCM2835 and BCM2836 devices which lack this boot code."
:grey_question: 
Although even that will need tidying up a bit in the copy-edit stage.

Reply:
I've just 'completely' rewritten that part based on your input.

I agree since it wasn't a very clear message, edited.